src/app/workspace/organization/users/user/organization-user.component.html

```diff
@@ -0,0 +1,67 @@
+<td class="px-1">
+  <i class="fa fa-caret-right large-text text-success {{ expanded ? 'fa-rotate-90' : null }}"
+     aria-hidden="true"
+     title="{{ expanded ? 'Close' : 'Open' }}"
+     i18n-title="@@organizations.user.expand"
+     (click)="expandedChange.emit()"></i>
+</td>
+
+<td class="px-1">
+  <input class="form-control edit-border"
+         type="text"
+         tabindex="1"
+         placeholder="Username"
+         [(ngModel)]="user.name"
```

Comment:
We must NEVER EVER directly modify the objects in the store! Otherwise the whole purpose of the store is useless and we will have an application full of strange bugs.